### DISCUSSION TITLE
Can variables be used when creating Registry values through the Intune Tools?

### DISCUSSION THREAD

**Discussion by theyssef** [Micke-K/IntuneManagement#66] [Q&A]
Hello All,

I'm looking to create Registry entries with the Intune Tools view to enable folder redirection.
Is this supported? And if so should i use: %USERNAME% or the powershell variant: $env:USERNAME ?

![image](https://user-images.githubusercontent.com/39700794/178214080-c8ececfd-1a10-4596-8c7f-e2279815d9d4.png)

Thanks!

Filip

**Comment by Micke-K** [comment]
Good question.
I have no clue but I do not think it is possible and if it is, it would not be $env:USERNAME. It's not added to the registry on the client side y a PowerShell script.
If you use variables, they will most likely end up with it in the name eg.
\\fileserver\Homes\%USERNAME%\...
Could be that the process using the value expands variables. So it might work even with the variable in the registry.
Just try and see what happens. 

However, if you plan to use it to redirect user folder there are probably way better options. Does this actually workif you do it manually on a device. I've only redirected folders to OneDrive in the last few years. GPOs before that to home drive.
